features/unattached_commands.feature

```diff
@@ -56,6 +56,18 @@ Feature: Command behaviour when unattached
         """
         -32768 <esm-apps-url> <release>-apps-security/main amd64 Packages
         """
+        When I run `python3 /usr/lib/ubuntu-advantage/ua_update_messaging.py` with sudo
+        And I run `run-parts /etc/update-motd.d/` with sudo
+        Then if `<release>` in `xenial` and stdout matches regexp:
+        """
+        \* Introducing Extended Security Maintenance for Applications.
+        Receive updates to over 30,000 software packages with
+        your Ubuntu Advantage subscription. Free for personal use.
+
+          https:\/\/ubuntu.com\/advantage
+
+        UA Infra: Extended Security Maintenance \(ESM\) is not enabled.
+        """
```

Comment:
I think its the indentation that this one is tripping over, maybe something like this would work?
```
        \* Introducing Extended Security Maintenance for Applications.
          Receive updates to over 30,000 software packages with
          your Ubuntu Advantage subscription. Free for personal use.

            https:\/\/ubuntu.com\/advantage

        UA Infra: Extended Security Maintenance \(ESM\) is not enabled.
```

Reply:
thanks used better whitespace match with a leading " +" in most lines.